Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -18,8 +18,8 @@ public class NetworkClient
         [Obsolete("Use NetworkClient directly. Singleton isn't needed anymore, all functions are static now. For example: NetworkClient.Send(message) instead of NetworkClient.singleton.Send(message).")]
         public static NetworkClient singleton = new NetworkClient();
 
-        [Obsolete("Use NetworkClient.singleton instead. There is always exactly one client.")]
-        public static List<NetworkClient> allClients => new List<NetworkClient>{singleton};
+        [Obsolete("Use NetworkClient directly instead. There is always exactly one client.")]
+        public static List<NetworkClient> allClients => new List<NetworkClient>();
```

Comment:
shouldn't we keep the allClients[0] for the sake of still being able to use .allClients while obsoleted?

Reply:
This PR was about changing the comment in the obsolete attribute.  If you'd rather this be coded differently, then close this.

Reply:
I am fine with the comment.
but you also changed the code:
![image](https://user-images.githubusercontent.com/16416509/55080188-fad71d00-509d-11e9-8364-f5855297c406.png)
which would make .allClients pointless. before your change, allClients[0] still pointed to the one and only NetworkClient :)

Reply:
Oh crap - old eyes - parens vs curlies - fixed